test/evaluation/test_evaluator.py

```diff
@@ -590,16 +590,16 @@ def test_metrics_multivariate(
 
 
 def test_evaluation_with_QuantileForecast():
-    start = "2012-01-01"
-    target = [2.4, 1.0, 3.0, 4.4, 5.5, 4.9] * 10
```

Comment:
I changed this so the OWA metric calculation doesn't fail here, which requires some training samples.